.github/actions/maven-setup/action.yml

```diff
@@ -2,10 +2,10 @@ name: "Setup Maven"
 description: "Setup JDK 17, maven cache, and JFrog credentials"
 
 inputs:
-  jfrog_user:
+  jfrog-user:
```

Comment:
the convention for inputs in github actions is using `-` as delimiter. this does mean we need to change all the services using this but i think that's fine.